packages/ui/src/components/input/input.tsx

```diff
@@ -0,0 +1,45 @@
+import { ChangeEvent, forwardRef } from "react";
+import { twMerge } from "tailwind-merge";
+
+interface InputProps {
+  className?: string;
+  id: string;
+  type?: string;
+  value?: string;
+  onChange?: (value: string) => void;
+  onBlur?: () => void;
+}
+
+export default forwardRef<HTMLInputElement, InputProps>(
+  function Input(props, ref) {
+    const { className, id, onBlur, onChange, type, value } = props;
```

Comment:
Many of these `props` are optional in `InputProps` but don't have default values. We can either make these required (preferred) or provide default values.